tests/conftest.py

```diff
@@ -397,3 +470,19 @@ def mock_repo_by_name(owner, name):
                 gh.init_account()
             db.session.expire(remote_token.remote_account)
             yield mock_api
+
+
+@pytest.yield_fixture
+def cli_run(app):
+    """Fixture for CLI runner function.
+
+    Returns a function accepting a single parameter (CLI command as string).
+    """
+    runner = CliRunner()
+    script_info = ScriptInfo(create_app=lambda info: app)
+
+    def run(command):
```

Comment:
Yes, but this utility function it's more intuitive when I can type a single string instead of a list.
But I guess I should use ``shlex.split(command)``, so ``'cmd "arg in quotes"'`` properly splits to ``['cmd', 'arg in quotes']``.